### PR TITLE
Send report at the end of the standup

### PR DESCRIPTION
**Pull request by santiagodoldan**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ApplicationMailer < ActionMailer::Base
+
+  default from: 'your-email@domain.com'
```
Comment (santiagodoldan) [inline app/mailers/application_mailer.rb]: @timchilcott what should we put here?

File: app/views/layouts/mail.html.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+!!!
+%html{ lang: "en" }
+  %head
+    %title Standup Bot by So Fetch
```
Comment (santiagodoldan) [inline app/views/layouts/mail.html.haml]: @timchilcott here? not sure if this appears somewhere in the email :P

File: app/views/standup_mailer/today_report.html.haml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+%div{ style: 'font-size: 30px; text-align: center; margin-bottom: 30px;' }
+  = "Standup of #{Time.now.strftime('%a, %d %B %Y').html_safe}"
```
Comment (santiagodoldan) [inline app/views/standup_mailer/today_report.html.haml]: @timchilcott this shows Standup of Sun, 10 January 2016, is that ok?

File: app/mailers/application_mailer.rb (hunk 1 above)
Comment (timchilcott) [inline app/mailers/application_mailer.rb]: Let's use standupbot@sofetch.io

File: app/views/layouts/mail.html.haml (hunk 2 above)
Comment (timchilcott) [inline app/views/layouts/mail.html.haml]: I like that.

File: app/views/standup_mailer/today_report.html.haml (hunk 3 above)
Comment (timchilcott) [inline app/views/standup_mailer/today_report.html.haml]: Let's format it it Standup of Sunday, January 10, 2016.

**Pull request merged**